src/parse_entity_sp_pl_cy.c

```diff
@@ -16,7 +16,7 @@ void	ft_parse_sphere(char *line, t_hittable *sphere, size_t id)
 {
 	t_sphere	*params;
 
-	sphere->id = id;
+	sphere->id = id + 1;
```

Comment:
Plus one due to the lights also having an ID now?

Reply:
No it's for the menu. In the menu the ID of the hittable is shown. It starts counting at 1, since it is more human like (three objects: 1,2,3)
The ID is (was?) never used anywhere else, so I thought I update it right here, and don't add 1 every time it gets put to screen.
The lights also start with 1, whereas ambient light has ID 0 (since its special as its not a moveable light).

Could also be reset, so that the first object has ID zero.